tokenizer.js

```diff
@@ -29,7 +29,9 @@
 			source: null,
 			allowUnknownTags: true,
 			numToSuggest: 5,
-			onclick: null
+			onclick: null,
+			allowDuplicates: false,
+			suggestDuplicates: false
```

Comment:
let's just use the `allowDuplicates` option, and automatically not suggest duplicates if they're not allowed.